internal/agentdeployer/factory.go

```diff
@@ -80,19 +76,16 @@ func Factory(options FactoryOptions) (AgentDeployer, error) {
 		// FIXME: this docker-compose scenario contains both agent and service
 		return nil, nil
 	case "k8s":
-		if _, err := os.Stat(agentDeployerPath); err == nil {
```

Comment:
It should not be needed to check that this folder exists, WDYT ? @jsoriano 
`findAgentDeployer` method returns already the folders found under `_dev/deploy`

Reply:
Yeah, looks correct, and indeed `agentDeployerPath` is not used anymore here after this change.